src/data_collectors.py

```diff
@@ -0,0 +1,71 @@
+import abc
+from concurrent.futures import Future
+from functools import partial
+import requests
+import re
+
+from constants import TIMEOUT_RSS_REQUEST
```

Comment:
remove unnecessary line break
sort imports

Reply:
https://peps.python.org/pep-0008/#imports

Imports should be grouped in the following order:

   1.  Standard library imports.
   2. Related third party imports.
   3. Local application/library specific imports.